applications/FSIApplication/tests/convergence_accelerator_spring_MPI_test.py

```diff
@@ -165,8 +163,7 @@ def test_accelerator(self,accelerator_settings,force1,force2,solution):
         res_norm = self.ComputeResidualNorm(residual)
 
         while (nl_it <= self.accelerator_iterations):
-
-            print(mpi.rank,": Iteration: ", nl_it," residual norm: ", res_norm, file=sys.stderr)
+            KratosMultiphysics.Logger.PrintInfoOnAllRanks('', str(KratosMultiphysics.DataCommunicator.GetDefault().Rank()) + ": Iteration: " + str(nl_it) + " residual norm: " + str(res_norm) + file=sys.stderr)
```

Comment:
I am quite sure `PrintInfoOnAllRanks` also prints the rank. Have you tried?
```suggestion
            KratosMultiphysics.Logger.PrintInfoOnAllRanks('', "Iteration: {}, residual norm: {}".format(nl_it, res_norm))
```

I hightly recommend embracing the power of [format](https://pyformat.info/), then there is  no more need for `"xxx" + str(some_int) + "vvv"`

Reply:
No, I didn't. I fully trust you on this.